frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -302,28 +302,34 @@ export const ImageManifestVulnList: React.FC<ImageManifestVulnListProps> = (prop
 };
 
 export const ImageManifestVulnPage: React.FC<ImageManifestVulnPageProps> = (props) => {
+  const { showTitle = true, hideNameLabelFilters = true } = props;
+  const namespace = props.namespace || props.match.params.ns || props.match.params.name;
   return (
     <MultiListPage
```

Comment:
We can easily use `ListPage` component here. Since it's only taking one type of resource `ImageManifestVulnModel`. Namespace condition through props and params is handled internally in the `ListPage` component.

Reply:
Even in `ListPage` component, we have to check the namespace condition and pass it. As this component is used in both the perspective and params name is not the same in both the perspective.